site/docs/v1/tech/apis/users.adoc

```diff
@@ -211,7 +211,7 @@ include::docs/v1/tech/apis/_user-response-body.adoc[]
 include::docs/v1/tech/apis/_generic-update-explanation-fragment.adoc[]
 :capitalized_object_name!:
 
-If you specify a new password for the User, it will be encrypted and stored. However, if you do not provide a new password, the User's old password will be preserved. This is the only field that is merged during an update using the `PUT` method.
+The one exception to this is with the password field. If you do not specify a password using the `PUT` method, the user's current password will be preserved and will not be deleted. A user's password will only be overwritten if you provide a new password.
```

Comment:
Did you mean to remove the previous copy? "The one exception to this.." seems like it should come after a previous statement which looks to be missing.